pkg/controller/actuator.go

```diff
@@ -17,13 +17,16 @@ package controller
 import (
 	extensionscontroller "github.com/gardener/gardener/extensions/pkg/controller"
 	"github.com/gardener/gardener/extensions/pkg/controller/network"
+	"github.com/pkg/errors"
 
 	"github.com/go-logr/logr"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/serializer"
 	"k8s.io/client-go/rest"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/log"
+
+	gardenerkubernetes "github.com/gardener/gardener/pkg/client/kubernetes"
```

Comment:
Please, rearrange the imports
1. golang packages
1. `github.com/gardener/` packages
1. third party packages

Reply:
Sure, we need an auto linter for gardener :D